.build/config/cassandra_ci.yaml

```diff
@@ -0,0 +1,342 @@
+    multiplex: 25
+    env:
+      DTEST_ARGS: "--use-vnodes --num-tokens=${NUM_TOKENS} --execute-upgrade-tests --execute-upgrade-tests-only --upgrade-target-version-only --upgrade-version-selection all"
+  
+  - job: dtest-upgrade-large
+    parent: dtest
+    description: Python based multi-node distributed upgrade tests flagged as too large for the regular upgrade suite
+    env:
+      DTEST_ARGS: "--use-vnodes --num-tokens=${NUM_TOKENS} --execute-upgrade-tests --execute-upgrade-tests-only --upgrade-target-version-only --upgrade-version-selection all --only-resource-intensive-tests --force-resource-intensive-tests"
+  
+  #-----------------------------------------------------------------------------
+  # Longer running soak / burn / benchmarking tests
+  #-----------------------------------------------------------------------------
+  - job: stress-test
+    runtime_env: jvm
+    resources: {cpu=4, memory=1g, storage=5g}
+    description: Runs stress tests
+
+  - job: fqltool-test
+    runtime_env: jvm
+    resources: {cpu=4, memory=1g, storage=5g}
+    description: Tests to ensure cassandra-stress is operating correctly
+  
+  - job: microbench
+    runtime_env: jvm
+    resources: {cpu=4, memory=1g, storage=5g}
+    description: Microbenchmarks executing specific portions of the codebase
+  
+  - job: test-burn
+    runtime_env: jvm
+    resources: {cpu=4, memory=16g, storage=5g}
+    description: Long running and fuzz tests
+    test_timeout: 60000000
+
+  - job: long-test
+    runtime_env: jvm
+    resources: {cpu=4, memory=6g, storage=10g}
+    description: Execute long-running tests that are beyond the normal suite timeout
+    test_timeout: 600000
+    multiplex: 100
+
+  - job: cqlsh-test
+    runtime_env: jvm
+    resources: {cpu=4, memory=1g, storage=5g}
+    description: Subset of tests exercising cqlsh only
+    cmd: ./pylib/cassandra-cqlsh-tests.sh $(pwd)
+
+#-----------------------------------------------------------------------------
+# PARAMETERS
+#-----------------------------------------------------------------------------
+# Since JVM tests are generally quicker, we can afford to multiplex across more params
+jdk_parameters:
+  - param: number
+    values:
+      - 11
+      - 17
+  - param: offheap
+    values:
+      - none
+      - objects
+      - all
+  - param: compression
+    values:
+      - on
+      - off
+  - param: memtables
+    values:
+      - trie
+      - btree
+  - param: sstables
+    values:
+      - bigtable
+      - tries
+
+# We can't afford to re-run all the python dtests across all the different
+# possible combinations above. We stick with just JDK as defined above and
+# vnode vs. novnode.
+python_parameters:
+  - params: num_tokens
+    values:
+      - 0
+      - 16
+
+#-----------------------------------------------------------------------------
+# Commands that can be pulled and run in pipelines
+#-----------------------------------------------------------------------------
+commands:
```

Comment:
I think I know where you copy from xD...

this is dead code

Reply:
Hah. Well, this was still in the circle config so I assumed it was something we relied on and could make use of. Very happy to delete this.